# The goal, to be sent by the client
sensor_msgs/Image template
---
# The result, to be sent by the server upon completion
bool is_complete
---
# The feedback, to be sent periodically by the server during execution
int32 nb_correspondences